### PR TITLE
add mailto to email link so it opens mail client

### PR DESCRIPTION
**Pull request by Jimeh87**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ ACME Corporation is one of the most...
 
 ## How to apply
 
-Please send an email to [email@email.com](email@email.com) with your resume, use 'Node Developer' as subject
+Please send an email to [email@email.com](mailto:email@email.com) with your resume, use 'Node Developer' as subject
```
Comment (LukeTowers) [inline .github/issue_template.md:62]:
```suggestion
Please send an email to [email@email.com](mailto:email@example.com) with your resume, use 'Node Developer' as subject
```

Comment (Jimeh87) [reply .github/issue_template.md:62]: I'm not quite sure I understand

Comment (LukeTowers) [reply .github/issue_template.md:62]: documentation should always refer to example.com as that is an officially reserved name specifically for use as a placeholder in documentation / content templates. email.com is a real domain owned by someone.

**Review by ryan4664 — APPROVED** [review]

**Comment by ryan4664** [comment]
LGTM!!

**Comment by LukeTowers** [comment]
+1

**Comment by Jimeh87** [comment]
@DJCrossman - @Sam-Dietrich's place of work hasn't been able to hire anyone since Feb 28th.

**Comment by DJCrossman** [comment]
After further review, I believe this will be an excellent addition to the template.

**Comment by ryan4664** [comment]
David for president

**Pull request merged**